### ISSUE TITLE
Question: Enable OpCache Preloading with php7.4+

### ISSUE THREAD

**Issue by MarkErik** [martialblog/docker-limesurvey#95]
Has anyone enabled OpCache Preloading with their Limesurvey configuration?

I am looking to improve the performance of Limesurvey, and from what I can gather via Google, that OpCache Preloading can have a positive impact on performance (at the expense of increased memory usage).

However, there doesn't seem to be any information as to how to configure it with Limesurvey, and I was hoping someone here has tried it :)

**Comment by martialblog** [comment]
Hi there, haven't worked with OpCache before. Have you asked in the LS forum? There's probably a larger user community there.

If we need some adjustmenst/enhancements in the Dockerfile here, feel free to open a PR. 

Cheers
Markus

**Comment by MarkErik** [comment]
Thanks for the quick response.

I had found a discussion between Limesurvey devs on their official feature/bug tracker, dating all the way back to 2019, however nothing came of it, and it was closed recently with no explanation.

As you suggest, I will post in the LS forum.

If I can get something basic working that would benefit the out-of-the-box experience for this project, I'll for sure add it as a PR.

Thanks!
